src/Middleware/CORS/src/PublicAPI.Shipped.txt

```diff
@@ -2,8 +2,6 @@
 ~Microsoft.AspNetCore.Cors.CorsPolicyMetadata.CorsPolicyMetadata(Microsoft.AspNetCore.Cors.Infrastructure.CorsPolicy policy) -> void
 ~Microsoft.AspNetCore.Cors.CorsPolicyMetadata.Policy.get -> Microsoft.AspNetCore.Cors.Infrastructure.CorsPolicy
 ~Microsoft.AspNetCore.Cors.EnableCorsAttribute.EnableCorsAttribute(string policyName) -> void
-~Microsoft.AspNetCore.Cors.EnableCorsAttribute.PolicyName.get -> string
-~Microsoft.AspNetCore.Cors.EnableCorsAttribute.PolicyName.set -> void
```

Comment:
Odd, looks like we somehow had a fair number of true duplicate lines in this file. How did we get away w/ that❔

Reply:
Because the analyzer sucks.